### PR TITLE
New version: Keybase.Keybase version 6.2.4

### LINKED ISSUE #135227: [Update Request]: Keybase v6.2.4
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

Keybase.Keybase

### Package Version

6.2.4

### Please describe the changes you would like to see

Source release https://github.com/keybase/client/releases/tag/v6.2.4
Binary release https://s3.amazonaws.com/prerelease.keybase.io/windows/Keybase_6.2.4-20231231194644%2Bae7e4a1c15.amd64.msi

### PR DESCRIPTION
**Pull request by orgcontrib**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/135242)

- Resolve #135227

### CODE REVIEW

**Comment by orgcontrib** [comment]
@microsoft-github-policy-service agree

**Comment by orgcontrib** [comment]
Resolve #135227

**Comment by orgcontrib** [comment]
> Resolve #135227

Comment appended to the main block above 👆🏼

File: manifests/k/Keybase/Keybase/6.2.4/Keybase.Keybase.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: '{98F546BD-1F4A-4D4C-A48D-E3200CB5D183}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://s3.amazonaws.com/prerelease.keybase.io/windows/Keybase_6.2.4-20231231194644%2Bae7e4a1c15.amd64.msi
```
Comment (stephengillie) [inline manifests/k/Keybase/Keybase/6.2.4/Keybase.Keybase.installer.yaml:19]:
```suggestion
  InstallerUrl: https://prerelease.keybase.io/windows/Keybase_6.2.4-20231231194644%2Bae7e4a1c15.amd64.msi
```

**Comment by stephengillie** [comment]
Hi @orgcontrib,

> This PR's version number 6.2.4 has 3 parameters (sets of numbers between dots - major, minor, etc), which is less than the current manifest's version 6.1.0.4, which has 4 parameters.

Is this intentional?

(Automated response - build 629)

**Comment by orgcontrib** [comment]
> Hi @orgcontrib,
> 
> > This PR's version number 6.2.4 has 3 parameters (sets of numbers between dots - major, minor, etc), which is less than the current manifest's version 6.1.0.4, which has 4 parameters.
> 
> Is this intentional?
> 
> (Automated response - build 629)

Yes and No. I want to keep it simple, and since this is a community effort, SymVer or SemVer matters less for me as long as there is no direct integration between upstream repo and this one. I'm a simple user of this app, and the main reason why I decided to contribute to its pkg refresh here is the recent problem with an expired x509 certificate.

**Comment by stephengillie** [comment]
Hi @orgcontrib,

> This PR's version number 6.2.4 has 3 parameters (sets of numbers between dots - major, minor, etc), which is less than the current manifest's version 6.1.0.4, which has 4 parameters.

Is this intentional?

(Automated response - build 643)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**